### DISCUSSION TITLE
Nav Menu with only 2 groups

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#1962] [Q&A] [answered]
Hi!I am trying to create a side menu with two options to switch between the dashboard and a user page. I am trying to avoid using tabs to conserve space vertically. Is there a way to create navigation menu with only group names and not items?Simply have two options on the side in order to switch between two pages.

**Comment by mturoci** [comment, marked as answer]
It seems like at least 1 nav group is currently required. It's reasonable to want to render top-level nav items without a group though. Feel free to file a feature request.

**Reply by n-srinidhi** [reply]
@mturoci Thank you!
